pkg/contact/endpoint.go

```diff
@@ -0,0 +1,50 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package contact
```

Comment:
There probably should be two packages `satellite/contact` and `storagenode/contact` since their behavior and dependencies are different.